libraries/chain/controller.cpp

```diff
@@ -5514,6 +5514,13 @@ signed_block_ptr controller::fetch_block_by_id( const block_id_type& id )const {
    return signed_block_ptr();
 }
 
+std::vector<char> controller::fetch_serialized_block_by_id( const block_id_type& id )const { try {
```

Comment:
We don't need the `fetch_serialized_block_by_id`. If we do in the future, I think it should verify the retrieved serialized block is the correct block id instead of just assuming the block_num of the id will retrieve the correct block.

Reply:
See new PR https://github.com/AntelopeIO/spring/pull/803 that removes its usage.